### PR TITLE
WRESTLING REFORGED

### PR DESCRIPTION
**Pull request by Djiq**
## About The Pull Request

This PR buffs wrestling to be on par with sleeping carp when it comes to lethality, adds to the uplink and also adds coder sprites because the actions didnt have any.
## Why It's Good For The Game

Wrestling is a niche martial art that is considerably less lethal than sleeping carp but much much more fun, i buffed it a bit so it isnt literally the weakest martial art you can get and put it in the uplink. This is good because the martial art wasnt good enough to be admin only but was too weak for anyone to ever ask for it.

## Changelog
:cl:
add: Championship belt has been added to the syndicate uplink. 
/:cl:

### CODE REVIEW

**Comment by Tharcoonvagh** [comment]
time for some WRASSLIN

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -601,6 +601,15 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
 	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
 
+/datum/uplink_item/stealthy_weapons/championbelt
+	name = "Syndicate Championship Belt"
+	desc = "Syndicate were able to steal this precious belt from a known wrestler. It was enhanced with implant like technology \
+	allowing you to access wrestling moves of the famous wrestlers. The moves were also enhanced to be extra lethal."
+	item = /obj/item/storage/belt/champion/wrestling
+	cost = 16
+	surplus = 0
+	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
```
Comment (ATH1909) [inline code/modules/uplink/uplink_items.dm:611]: eh, nuke ops get access to CQC and krav maga anyway, why not let them have access to this, too?

Comment (EdgeLordExe) [reply code/modules/uplink/uplink_items.dm:611]: good idea

File: code/datums/martial/wrestling.dm [hunk 2]
```diff
@@ -17,6 +17,18 @@
 	var/datum/action/strike/strike = new/datum/action/strike()
 	var/datum/action/drop/drop = new/datum/action/drop()
 
+/datum/martial_art/wrestling/teach(mob/living/carbon/human/H, make_temporary)
+	. = ..()
+	ADD_TRAIT(H,TRAIT_STUNRESISTANCE,type)
+	ADD_TRAIT(H,TRAIT_PUSHIMMUNE,type)
+	ADD_TRAIT(H,TRAIT_IGNOREDAMAGESLOWDOWN,type)
```
Comment (Iamgoofball) [inline code/datums/martial/wrestling.dm:24]: Jesus christ

**Review by Iamgoofball — CHANGES_REQUESTED** [review]

File: code/datums/martial/wrestling.dm (hunk 2 above)
Comment (EdgeLordExe) [reply code/datums/martial/wrestling.dm:24]: stun resistance might be a bit much i agree

**Comment by optimumtact** [comment]
I would prefer that sleeping carp be nerfed instead

**Comment by necromanceranne** [comment]
Scarp: telebaton hands
Wrestling: literally uncounterable series of attacks that hardstun you throughout the chain and can knock you out.

Maybe I'm insane but I feel like I defanged the moveset itself well enough that it isn't remotely as lethal as something that is more equivalent to CQC. It wasn't the moveset that made Sleeping Carp hard to fight before shaps nerfed it. Maybe if older scarp was still in the game this would be equal, but do we really want hardstun hands 2.0?

**Comment by optimumtact** [comment]
I want to remove all the martial arts, they're cringe

**Comment by EdgeLordExe** [comment]
hm actually yeah the moveset is busted, but a single shove and you are dead, i think it may be better off as a wizard martial art since we dont have any and hey it might be fun

File: code/datums/martial/wrestling.dm (hunk 2 above)
Comment (Iamgoofball) [reply code/datums/martial/wrestling.dm:24]: i didnt click stun resistance, i clicked the ignore damage slowdown

**Comment by Doosteroni** [comment]
>  as a wizard martial art

Oh no brother, how am I supposed to  SNAP INTO A SLIM JIM if you gonna give it to wiznerds? Carp literally makes you immune to ranged damage. You can easily eliminate a wrestler at range. If at the very least can you add a pre-buff wrestling if your gonna ruin my one and only dream in life which is to wrestle without an admin on because I only TC trade for wrestling brother

**Comment by Doosteroni** [comment]
Honestly I don't think it really warrants being a wizard martial art. You can easily tell who is a wrestler because of their bigass belt. A bartender or sec can easily kill off a wrestler.

**Comment by Ryll-Ryll** [comment]
>hm actually yeah the moveset is busted, but a single shove and you are dead

>ADD_TRAIT(H,TRAIT_PUSHIMMUNE,type)

meme

**Comment by EdgeLordExe** [comment]
I think it can stay as a 16 tc martial art, the only thing it has going for it are stuns, and the damage it deals isnt that big.

**Comment by GuyonBroadway** [comment]
>I want to remove all the martial arts, they're cringe

Make ranged combat more interesting and then we'll talk. 

Also make killing people more interesting while you are at it

**Comment by ExcessiveUseOfCobblestone** [comment]
all this does is just up the damage of this cringe martial art and make it available.

This is the martial art with TP dropkick, instant grab into slam spam correct?

**Comment by Rohesie** [comment]
Sorry, we want to diminish/substitute martial arts, not expand on them.

**Pull request closed without merging**